scripts/format_code.py

```diff
@@ -28,6 +28,7 @@
      because -noformat_file was set.
    2: The application wasn't configured properly and could not execute.
 """
+import difflib
```

Comment:
is this in python by default or do we need to instruct users to install it for this script?

Reply:
It's part of the Python standard library.

https://docs.python.org/3/library/difflib.html